tox.ini

```diff
@@ -38,7 +38,7 @@ passenv =
     SSH_AUTH_SOCK
     TERM
 setenv =
-    ANSIBLE_CONFIG={toxinidir}/dev/null
+    ANSIBLE_CONFIG={toxinidir}/.ansible.cfg
```

Comment:
This this really needed, why? Seems unrelated.

Reply:
If ANSIBLE_CONFIG points to a non-existent path, then the value is ignored and Ansible falls back to any that exist on the local system. This causes local runs of tests to fail, at least on my system, because of some of my ~/.ansible.cfg settings.

True, this isn't strictly necessary, but it was necessary to pass in my environment.